app/client/ui/PageWidgetPicker.ts

```diff
@@ -260,8 +261,7 @@ export function buildPageWidgetPicker(
     dom.create(PageWidgetSelect,
       value, tables, columns, onSaveCB, behavioralPromptsManager, options),
 
-    // gives focus and binds keydown events
-    (elem: any) => { setTimeout(() => elem.focus(), 0); },
+    elem => { FocusLayer.create(ctl, {defaultFocusElem: elem, pauseMousetrap: true}); },
```

Comment:
Is this needed here?

Reply:
> Is this needed here?

[This test](https://github.com/gristlabs/grist-core/actions/runs/10641941256/job/29503756696) was failing regularly without it.